tests/cypress/page-object/exportDialog.ts

```diff
@@ -1,19 +1,19 @@
 import {BaseComponent, Button, Dropdown, getComponentByAttr} from '@jahia/cypress';
 
 export class ExportDialog extends BaseComponent {
-    static defaultSelector = 'button[data-sel-role="breadcrumb-item"]';
+    static defaultSelector = '[data-cm-role="export-options"]';
 
     export(downloadsFolder:string, workspace:string = 'Staging content only', exportFormat: string = 'zip') {
         const dialog = getComponentByAttr(BaseComponent, 'data-cm-role', 'export-options');
```

Comment:
sorry one more minor thing, but I think this line should be safe to remove. 

I don't think we use `dialog` anywhere now.

Reply:
we use it here  getComponentByAttr(Dropdown, 'data-cm-role', 'select-workspace', dialog).select(workspace); 
(line 10)

Reply:
ah, thanks. in this case `getComponentByAttr` has `parent` param of type `BaseComponent`. 

Here, we can pass `this` instead of `dialog` - `getComponentByAttr(Dropdown, 'data-cm-role', 'select-workspace', this).select(workspace);`